Example/ViewController.swift

```diff
@@ -62,7 +62,7 @@ struct EstablishmentsResponse {
 }
 
 extension EstablishmentsResponse: Parsable {
-    static func parse(from data: Data?, status: Int, headers: [String : String]?, errorParser: ErrorParsing.Type?, userInfo: [String : Any]?) -> Result<EstablishmentsResponse> {
+    static func parse(from data: Data?, status: Int, headers: [String: String]?, errorParser: ErrorParsing.Type?, userInfo: [String: Any]?) -> Result<EstablishmentsResponse> {
```

Comment:
Does this indicate my swiftformat setup isn't right?

Reply:
No, just that I hadn't run it on the sample...